src/components/products-page/products-item-card.styles.scss

```diff
@@ -0,0 +1,37 @@
+.card-background {
+    background-color: #F8F5F2;
```

Comment:
the background has a 0.9 opacity.
it kind of tricks you in adobe XD coz it has a opacity parameter set to 1, but the opaque hex codes for colors only have 6 characters (2 hex numbers, called octets, for red, green, and blue), but in the design it has 8 hex numbers. the last 2 are for opacity. not sure if 8 hex numbered values are allowed in CSS3, so if its not you can just use RGBA.

Reply:
yeh try to follow the design as close as possible and just keep an eye out for transparent elements I guess